build.sbt

```diff
@@ -123,7 +123,9 @@ lazy val testchipip = (project in file("generators/testchipip"))
   .settings(commonSettings)
 
 lazy val chipyard = conditionalDependsOn(project in file("generators/chipyard"))
-  .dependsOn(boom, hwacha, sifive_blocks, sifive_cache, utilities, sha3, gemmini, icenet, tracegen)
+  .dependsOn(boom, hwacha, sifive_blocks, sifive_cache, utilities,
+    sha3,
```

Comment:
I feel like there should be a comment here about this, because someone is definitely going to add another dependency at some point and merge the lines (and then fail CI and not understand why....)

Reply:
For those of us not in the loop, why does this fail?

Reply:
To setup the repo for tutorial mode (delete SHA3 to walk attendees through integrating it), we need to patch away lines referencing sha3.

If sha3 is not on its own line, the patch will fail.

Reply:
+1 on sticking a comment on this line